esp/scm/esploggingservice.ecm

```diff
@@ -115,7 +115,7 @@ TransactionSeedResponse
     bool SeedAvailable;
 };
 
-ESPService [requiredaccess("NONE"),noforms] WsLogService
+ESPService [autofeatureauth("NONE"),noforms] WsLogService
```

Comment:
@wangkx does it make sense to perform feature level auth on any/all of these methods?